data/json/npcs/talk_tags.json

```diff
@@ -1095,6 +1110,7 @@
       "Pass some ethanol, I need to power my ethanol burner.",
       "Waiter!  I need a refill, my ethanol burner is running out of charge!",
       "I require ethanol for my internal power supply.  Anything on you?",
+      "I need some ethanol to power up my CBMs.",
```

Comment:
What if there's only one CBM?

Reply:
The <need_batteries> section right above this has similar dialogue. Is that also an error?